apps/extension/src/ui/apps/dashboard/routes/Settings/Mnemonics/MnemonicBackupModal/context.ts

```diff
@@ -0,0 +1,52 @@
+/* eslint-disable react/display-name */
```

Comment:
I think this can be removed from this file with no other change?

Reply:
👍